gen/generator.toml

```
@@ -112,6 +112,24 @@ auto_mutability_blacklist = []
 # opposite to `auto_mutability_blacklist` and has a higher priority
 auto_mutability_whitelist = []
 
+# if set to "raw", extract and dump raw c comment;
+# if set to "doxygen", parse and format doxygen comment.
+# note: by default, Clang only parses doxygen comment, pass `-fparse-all-comments` to Clang in order to parse non-doxygen comments.
+extract_c_comment_style = "disable"
+
+# if set to true, single line comment will be printed as """comment""" instead of """\ncomment\n"""
+fold_single_line_comment = true
```

```suggestion
fold_single_line_comment = false
```

to match the default behavior in the code: `fold_single_line_comment = get(options, "fold_single_line_comment", false)`

---

Is this file the default behavior or only what is used to generate LibClang.jl? I find that a few other options are not the default behavior, for example, "use_deterministic_symbol" defaults to false.

---

For those options that are not currently being used for generating libclang C bindings, so better to make them consistent with the default behavior in the code. But apparently, I did it wrong in the case of `auto_mutability`...